### ISSUE TITLE
Why is `import/unambiguous` included in recommended config but not in warnings config?

### ISSUE THREAD

**Issue by briandipalma** [benmosher/eslint-plugin-import#1063]
Recommended includes `import/unambiguous` in it's warnings section; which causes a lot of noise unless you have a fully ES2015 modules code base, but it's not included in the warnings config. This seems inconsistent. I can create a PR to change one or the other. I'd prefer to remove it from recommended given it seems an issue that isn't live yet with no ESM support in node.

**Comment by ljharb** [comment]
I agree that it shouldn’t be recommended in any form, since unambiguous grammar was rejected by TC39.
